spring-cloud-gcp-samples/spring-cloud-gcp-sql-mysql-r2dbc-sample/README.adoc

```diff
@@ -0,0 +1,47 @@
+= Spring Cloud GCP MySQL R2DBC Sample
+
+This code sample demonstrates how to connect to a MySQL Google Cloud SQL instance using the link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql-mysql-r2dbc[Spring Cloud GCP MySQL R2DBC Starter].
+
+You will create an instance, a database within the instance, populate the database and then query it.
+
+== Setup
+
+image:http://gstatic.com/cloudssh/images/open-btn.svg[link=https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fspring-cloud-gcp&cloudshell_open_in_editor=spring-cloud-gcp-samples/spring-cloud-gcp-sql-mysql-r2dbc-sample/README.adoc]
+
+1. Follow https://cloud.google.com/sql/docs/mysql/quickstart[these instructions] to set up a Google Cloud Project with billing enabled.
+
+2. Create a Google Cloud MySQL instance from the https://console.cloud.google.com/sql/instances[Google Cloud Console Cloud SQL page].
+
+3. Still within the Google Cloud Console SQL page, create a new database in the instance from the "Databases" section.
+If you decided to set a root password for your instance footnoteref:[note,Google Cloud SQL allows for creating MySQL instances without a password,while still remaining safe,as the connection is using Cloud SDK authentication and encrypted communication: see docs on https://cloud.google.com/sql/docs/mysql/sql-proxy[Cloud SQL proxy] and https://cloud.google.com/sql/docs/mysql/connect-external-app#java[the java socketFactory],which is used in spring-cloud-gcp to create the connection!], then don't forget to take note of your root password, as you will need it in the next step!
+
+4. Open the link:src/main/resources/application.properties[application.properties] file and replace the following:
+- *[database-name]* with the name of the database you created in step 3.
+- *[instance-connection-name]* with the instance name of the MySQL instance you created in step 1.
```

Comment:
Add a bit of information -- "instance name (in the format `[GCP_PROJECT_ID]:[INSTANCE_REGION]:[INSTANCE_NAME]`)"